CMakeLists.txt

```diff
@@ -81,11 +81,17 @@ include_directories(${Boost_INCLUDE_DIRS})
 # find Python interpreter. Needed for tests, and best to enforce consistency anyway.
 find_package(PythonInterp QUIET)
 find_package(PythonLibs QUIET)
-option(BUILD_PYTHON "Build SIRF with python" ${PYTHONLIBS_FOUND})
-if (BUILD_PYTHON AND NOT ${PYTHONLIBS_FOUND})
-  message(FATAL_ERROR "Python enabled but not found")
-endif(BUILD_PYTHON AND NOT ${PYTHONLIBS_FOUND})
-if (BUILD_PYTHON)
+if (NOT PYTHONLIBS_FOUND)
+  message(STATUS "Pythonlibs not found (set PYTHON_EXECUTABLE if you want it)")
+  set(default_DISABLE_PYTHON ON)
+else()
+  set(default_DISABLE_PYTHON OFF)
+endif()
+
+option(DISABLE_PYTHON "Disable building SIRF python support" ${default_DISABLE_PYTHON})
```

Comment:
Could this be done with the following?
```
option(DISABLE_PYTHON "Disable building SIRF python support" NOT ${PYTHONLIBS_FOUND})
```
which would save creating the variable `default_DISABLE_PYTHON`

Reply:
nope. CMake doesn't do boolean operations. only inside `IF()`. surprised? So was I.

Reply:
huh, strange. PR is fine with me in that case.

Reply:
ah. maybe it can via [generator expressions](https://cmake.org/cmake/help/v3.13/manual/cmake-generator-expressions.7.html). It seems that `$<NOT:PYTHONLIBS_FOUND>` might work.

Reply:
seems it needs something like `$<IF:$<BOOL:PYTHONLIBS_FOUND>, OFF, ON>`. Not sure if that's clearer...

Reply:
Sure, fine to leave as is.